weather_dl/download_pipeline/clients.py

```diff
@@ -32,8 +32,7 @@
 from .config import Config, optimize_selection_partition
 from .util import retry_with_exponential_backoff
 
-warnings.simplefilter(
-    "ignore", category=urllib3.connectionpool.InsecureRequestWarning)
+warnings.simplefilter("ignore", category=urllib3.connectionpool.InsecureRequestWarning)
```

Comment:
I bet we'll have more files than this that needs formatting :)

Once we have pyink set up in our dev environments + CI, I recommend applying the formatter once to every tool (`weather_dl`, `weather_mv`, `weather_sp`).

Reply:
In the beginning I was afraid to do changes to whole directories. Sure if our approches work :)